templates/shared_services/firewall/terraform/locals.tf

```diff
@@ -15,4 +15,7 @@ locals {
   api_driven_network_rule_collection     = jsondecode(base64decode(var.api_driven_network_rule_collections_b64))
 
   firewall_policy_name = "fw-policy-${var.tre_id}"
+
+  default_firewall_sku   = "Standard"
+  effective_firewall_sku = coalesce(var.firewall_sku, local.default_firewall_sku)
```

Comment:
Why was this needed (vs using the default value)?

Reply:
It's to handle the FIREWALL_SKU parameter being present but its value being an empty string.  There was a situation where this occurred while I was testing... let me dig it out.

Reply:
Ah OK... yes... it was to handle the situation where people had not set `FIREWALL_SKU` as a GitHub Actions variable.  I suppose I could have defaulted the GitHub Action input parameter to 'Standard' but it felt more appropriate to keep the default value in a single place & closer to the actual firewall terraform - since there are multiple routes for the parameter being passed in (from GitHub Actions variables, from config.yaml, via the resource processor).

https://github.com/nwsde/nwsde-azuretre/blob/ad5ac13907869df2efb262dd0a6b993b7df2b504/.github/actions/devcontainer_run_command/action.yml#L128

https://github.com/nwsde/nwsde-azuretre/blob/ad5ac13907869df2efb262dd0a6b993b7df2b504/.github/actions/devcontainer_run_command/action.yml#L241

Reply:
ok. thanks for the explanation